### PR TITLE
[Tests-Only]Wisely used skeleton files in Favorites, Files, and FilesActionMenu suites

### PR DESCRIPTION
**Pull request by Talank**
## Description
refactor steps for user creation with appropriate skeleton files.
Following suites have been refactored: 
- webUIFavorites
- webUIFiles
- webUIFilesActionMenu

## Related Issue
- Part of https://github.com/owncloud/QA/issues/659

## How Has This Been Tested?
- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14272/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14274/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14325/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14342/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14342/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14385/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14385/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14387/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14385/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14387/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14423/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14423/

<p>







</p></details>

**Review by dpakach — COMMENTED** [review]
using `user has created file` will just create an empty text file. For the files tests, we should upload "real" files to make the tests more reliable IMO

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1

**Comment by Talank** [comment]
> using `user has created file` will just create an empty text file. For the files tests, we should upload "real" files to make the tests more reliable IMO

But again, uploading a file would magically put some contents in the file, and the person who read the feature file would not know what the content was. I mean the content is not clear from the scenario itself. IMO, we should minimize the uploading and rather create files with contents. 🤔 what do you think @individual-it

**Comment by dpakach** [comment]
But in a scenario, when you see the user has `testimage.jpg` then it should be an image file. not an empty text file with `.jpg` extension.

**Comment by individual-it** [comment]
for txt files its good to mention the content while uploading and if it relevant to the test, for other file types we have to upload the content from the filesToUpload folder. I agree with @dpakach don't create a "fake" file by just naming it `.jpg` upload a real file

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14525/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14525/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14527/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14527/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14532/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14532/

<p>







</p></details>

File: tests/acceptance/features/webUIFavorites/favoritesFile.feature [hunk 1]
```diff
@@ -6,12 +6,15 @@ Feature: Mark file as favorite
...
   Background:
-    Given user "Alice" has been created with default attributes
+    Given user "Alice" has been created with default attributes and without skeleton files
     And user "Alice" has logged in using the webUI
     And the user has browsed to the files page
 
   @smokeTest
   Scenario: mark files as favorites
+    Given user "Alice" has created file "data.tar.gz"
+    And user "Alice" has created file "data.zip"
```
Comment (saw-jan) [inline tests/acceptance/features/webUIFavorites/favoritesFile.feature:16]:
```suggestion
    And user "Alice" has uploaded file "data.zip" to "data.zip"
```

Comment (saw-jan) [reply tests/acceptance/features/webUIFavorites/favoritesFile.feature:16]: `has created file` step is best to use for `.txt` file

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14688/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14688/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14688/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14706/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14706/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14715/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14715/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14723/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14723/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14738/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14738/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14745/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14745/

<p>







</p></details>

File: tests/acceptance/features/webUIFiles/breadcrumb.feature [hunk 2]
```diff
@@ -4,11 +4,12 @@ Feature: access breadcrumb
...
 
   Background:
-    Given user "Alice" has been created with default attributes
+    Given user "Alice" has been created with default attributes and without skeleton files
 
 
   Scenario: Breadcrumb navigation should not happen on last segment
-    Given user "Alice" has created folder "simple-folder/subfolder"
+    Given user "Alice" has created folder "simple-folder"
+    And user "Alice" has created folder "simple-folder/subfolder"
```
Comment (swoichha) [inline tests/acceptance/features/webUIFiles/breadcrumb.feature:12]: IMO
```suggestion
    Given user "Alice" has created folder "simple-folder/subfolder"
```

File: tests/acceptance/features/webUIFiles/browseDirectlyToDetailsTab.feature [hunk 3]
```diff
@@ -5,7 +5,9 @@ Feature: browse directly to details tab
   So that I can see the details immediately without needing to click in the UI
 
   Background:
-    Given user "Alice" has been created with default attributes
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "simple-folder"
+    And user "Alice" has created file "simple-folder/lorem.txt"
```
Comment (swoichha) [inline tests/acceptance/features/webUIFiles/browseDirectlyToDetailsTab.feature:10]:
```suggestion
    And user "Alice" has created file "simple-folder/lorem.txt"
```

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14916/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14916/

<p>







</p></details>

File: tests/acceptance/features/webUIFiles/breadcrumb.feature (hunk 2 above)
Comment (Talank) [reply tests/acceptance/features/webUIFiles/breadcrumb.feature:12]: we can not create a subfolder and parent folder at the same time like that.

File: tests/acceptance/features/webUIFiles/browseDirectlyToDetailsTab.feature (hunk 3 above)
Comment (Talank) [reply tests/acceptance/features/webUIFiles/browseDirectlyToDetailsTab.feature:10]: same here

**Review by swoichha — COMMENTED** [review]
LGTM

**Review by saw-jan — COMMENTED** [review]
LGTM:+1:

**Review by individual-it — APPROVED** [review]
please rebase on master

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15075/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15079/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15085/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15109/

<p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15114/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15116/

<p>






</p></details>

**Pull request merged**